sklearn/impute.py

```diff
@@ -382,6 +399,9 @@ def transform(self, X):
             raise ValueError("X has %d features per sample, expected %d"
                              % (X.shape[1], self.statistics_.shape[0]))
 
+        if self.add_indicator:
+            X_trans = self.indicator_.transform(X)
```

Comment:
It feels weird to call this `X_trans` when the actual `X_trans` is named `X`.

Maybe rename this into `X_trans_indicator`